sockeye/beam_search.py

```diff
@@ -67,10 +68,17 @@ def encode_and_initialize(self, inputs: pt.Tensor, valid_length: Optional[pt.Ten
         states, predicted_output_length = self._model.encode_and_initialize(inputs, valid_length, self._const_lr)
         return states, predicted_output_length
 
+    def get_model(self):
```

Comment:
It seems you are working around the fact that we made the model a 'private' member of this class by naming it `_model`. You can change that if you want, then you don't need this getter (which isn't really pythonic either).
Alternatively you can do:
```
@property
def model(self):
  return self._model
```
and access via `inference.model`

Reply:
Done. Thanks!